### PR TITLE
add certs back

### PR DESCRIPTION
**Pull request by jordan-rash**
## Feature or Problem
Need to run CA Certs update for wadm to work with NGS

I dont have docker on this machine, so im letting CI sanity check it

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 FROM debian:bullseye-slim AS base
 
+RUN update-ca-certificates
+
```
Comment (brooksmtownsend) [inline Dockerfile:4]: Will this work if ca-certificates aren't installed? And don't we have to do this on the final container (e.g. after line 13)?

Comment (brooksmtownsend) [reply Dockerfile:4]: Sadly, no Docker in CI I don't think 😬

Comment (jordan-rash) [reply Dockerfile:4]: The first container is the base container, so running it in there should trickle into the others.....I need to go find another laptop to check it...ill report back

**Comment by jordan-rash** [comment]
This should work brooks

**Review by brooksmtownsend — APPROVED** [review]

**Pull request merged**